spec/type.dd

```diff
@@ -216,20 +216,20 @@ ulong  u4 = cast(long)-1; // ok, -1 can be represented in a ulong
 
 $(H2 $(LNAME2 bool, $(D bool)))
 
-    $(P The bool type is a 1 byte size type that can only hold the
-    value $(D_KEYWORD true) or $(D_KEYWORD false).
-    The only operators that can accept operands of type bool are:
-    $(CODE_AMP) $(CODE_PIPE) $(D ^) $(CODE_AMP)$(D =) $(CODE_PIPE)$(D =)
-    $(D ^=) ! $(CODE_AMP)$(CODE_AMP) $(CODE_PIPE)$(CODE_PIPE) $(D ?:).
-    A $(D bool) value can be implicitly converted to any integral type,
-    with $(D_KEYWORD false) becoming 0 and $(D_KEYWORD true) becoming 1.
-    The numeric literals 0 and 1 can be implicitly
-    converted to the bool values $(D_KEYWORD false) and $(D_KEYWORD true),
-    respectively.
-    Casting an expression to bool means testing for 0 or !=0 for
-    arithmetic types, and $(D_KEYWORD null) or !=$(D_KEYWORD null)
-    for pointers or references.
-    )
+$(P The bool type is a byte-size type that can only hold the value `true` or
```

Comment:
This hunk changes both content and whitespace, meaning it's very difficult to see just the differences in content.